.github/workflows/linting.yml

```diff
@@ -13,6 +13,8 @@ jobs:
       - name: Yamllint
         uses: karancode/yamllint-github-action@master
         with:
+          yamllint_strict: true
+          yamllint_format: parsable
```

Comment:
memo) The comments will be easier to read as follows.

### Before
![before](https://user-images.githubusercontent.com/13041216/226358561-77b25291-870a-402c-b139-ea48fe5a3d56.png)

### After
![after](https://user-images.githubusercontent.com/13041216/226358568-616ffb0d-ed37-4b49-a943-f7b8ffcd02de.png)